fido-2-derivation-nicolas.md

```diff
@@ -53,7 +56,7 @@ We should specify (independent of used crypto library) whether P256 seed is inte
 
 Outputs (sent to RP):
 
-- Credential ID: `[version, raw_credential_id, tag]` (1 + 32 + 32 = 65B)
+- Credential ID: version || tag || raw_credential_id || seed_generator_metadata
```

Comment:
An alternative might also be: `version || raw_credential_id || metadata || tag`.
- could set tag = `H(K2, version || raw_credential_id || metadata)` (advantage being don't have to "piece together" the inputs to the hash function, can just pass the slice)
- I first thought this would need a "length" byte/word, but it's not strictly necessary: tag is "last 16 bytes", version is first byte, raw_credential_id is bytes 2 to 17 (authenticator can simply treat metadata as "blog" in normal use)